src/pages/Yeonju/Login/LoginYeonju.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './LoginYeonju.scss';
+import { withRouter } from 'react-router-dom';
+
+class LoginYeonju extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idValue: '',
+      pwValue: '',
+      isActive: false,
+    };
+  }
+  goToMain = () => {
+    this.props.history.push('/mainyeonju');
+  };
+
+  handleIdInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+    console.log(this.state);
+  };
+
+  changeButton = () => {
+    const { idValue, pwValue } = this.state;
+    idValue.includes('@') && pwValue.length >= 5
+      ? this.setState({ isActive: true })
+      : this.setState({ isActive: false });
```

Comment:
this.setState를 제일 앞으로 보내고, 데이터 타입 활용하면 코드를 더 줄일 수 있어요! 🙂

```suggestion
    this.setState({ isActive : idValue.includes('@') && pwValue.length >= 5 })
```

Reply:
와우.. 꿀팁 감사합니다!